src/transformers/models/clip/modeling_clip.py

```diff
@@ -95,12 +95,12 @@ class CLIPOutput(ModelOutput):
     """
 
     loss: Optional[torch.FloatTensor] = None
-    logits_per_image: torch.FloatTensor = None
-    logits_per_text: torch.FloatTensor = None
-    text_embeds: torch.FloatTensor = None
-    image_embeds: torch.FloatTensor = None
-    text_model_output: BaseModelOutputWithPooling = None
-    vision_model_output: BaseModelOutputWithPooling = None
+    logits_per_image: Optional[torch.FloatTensor] = None
+    logits_per_text: Optional[torch.FloatTensor] = None
+    text_embeds: Optional[torch.FloatTensor] = None
+    image_embeds: Optional[torch.FloatTensor] = None
+    text_model_output: Optional[BaseModelOutputWithPooling] = None
+    vision_model_output: Optional[BaseModelOutputWithPooling] = None
```

Comment:
I don't think the `Optional` type is necessary for dataclass definitions, just function headers!

Reply:
revised